addons/mail/static/src/models/channel.js

```diff
@@ -136,13 +151,19 @@ registerModel({
             compute: '_computeCorrespondentOfDmChat',
             inverse: 'dmChatWithCurrentPartner',
         }),
+        displayName: attr({
+            compute: '_computeDisplayName',
+        }),
         id: attr({
             identifying: true,
         }),
         /**
          * States the number of members in this channel according to the server.
          */
         memberCount: attr(),
+        name: attr({
+            related: 'thread.name',
+        }),
```

Comment:
I don't like this kind of related, I'd rather see `thread.name` directly in code.